.github/workflows/system-tests-opensource.yml

```diff
@@ -17,34 +17,35 @@ name: System Tests Open Source
 on:
   push:
     branches:
-      - '.+-system-tests'
+      - '.+-os-system-tests'
 
-  schedule:
-
-    # * is a special character in YAML so you have to quote this string
-    # Run the system tests every 3 hours
-    - cron:  '0 */3 * * *'
+# disabled until we have a proper schedule definition
+#  schedule:
+#
+#    # * is a special character in YAML so you have to quote this string
+#    # Run the system tests every 3 hours
+#    - cron:  '0 */3 * * *'
 
   workflow_dispatch:
     inputs:
-      docker_registry:
-        description: 'Docker registry to pull images from (default: ghcr.io/, use registry.hub.docker.com/ for docker hub)'
-        required: true
-        default: 'ghcr.io/'
-      docker_repo:
-        description: 'Docker repo to pull images from (default: mlrun)'
-        required: true
-        default: 'mlrun'
+      pr_number:
+        description: 'PR number to run the system tests against (default: run on the latest commit in the development branch)'
+        required: false
+        default: ''
+      pytest_markers:
+        description: 'Pytest markers to run'
+        required: false
+        default: 'not enterprise and smoke'
```

Comment:
general question - are the used/supported markers written anywhere?

Reply:
u manually create markers  by using `pytest.markers.<X>` where X is any arbitrary marker